introduction.md

```diff
@@ -1,7 +1,6 @@
 I found Ruby better than GIT.
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?